### PR TITLE
FastScan GA

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

remove the experiment feature warning.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/11151

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by Lloyd-Pottiger — APPROVED** [review]

**Comment by shichun-0415** [comment]
@hongyunyan Seems these docs also need update:
1. [TiDB 功能概览-索引和约束](https://docs.pingcap.com/zh/tidb/dev/basic-features#索引和约束): Add the 7.0 column and change E to Y

image

2. [系统变量-tiflash_fastscan](https://docs.pingcap.com/zh/tidb/dev/system-variables#tiflash_fastscan-从-v630-版本开始引入): Change default value OFF to ON

image

3. [TiFlash v6.2 升级帮助(https://docs.pingcap.com/zh/tidb/dev/tiflash-620-upgrade-guide)

image

File: basic-features.md [hunk 1]
```diff
@@ -44,22 +44,22 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [使用 bucket 提高数据扫描并发度](/tune-region-performance.md#使用-bucket-增加并发) | E | E | E | N | N | N | N | N | N |
-| [不可见索引](/sql-statements/sql-statement-add-index.md)               |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
-| [复合主键](/constraints.md#主键约束)                                   |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [唯一约束](/constraints.md#唯一约束)                                   |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [整型主键上的聚簇索引](/constraints.md)                                |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    Y     |
-| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
-| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | N | N | N | N | N | N | N |
-| [外键约束](/constraints.md#外键约束) | Y | N | N | N | N | N | N | N |
+| 索引和约束                                                             | 6.7 | 6.6 | 6.5 | 6.1 |   5.4    |   5.3    |   5.2    |   5.1    |   5.0    |   4.0    |
```
Comment (shichun-0415) [inline basic-features.md:47]:
```suggestion
| 索引和约束                                                             | 7.0 | 6.6 | 6.5 | 6.1 |   5.4    |   5.3    |   5.2    |   5.1    |   5.0    |   4.0    |
```

File: tiflash-620-upgrade-guide.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ summary: 了解升级 TiFlash 至 v6.2 时的注意事项。
 
 > **注意：**
 >
-> - v6.2.0 新增了一项名为 [FastScan](/develop/dev-guide-use-fastscan.md) 的实验功能。该实验功能在牺牲强一致性保证的前提下可以大幅提升扫表性能。相关实验功能的形态和使用方式有可能在后续版本中发生变化。
+> - v6.2.0 新增了一项名为 [FastScan](/develop/dev-guide-use-fastscan.md) 的实验功能，该功能在 7.0 GA。该实验功能在牺牲强一致性保证的前提下可以大幅提升扫表性能。相关实验功能的形态和使用方式有可能在后续版本中发生变化。
```
Comment (shichun-0415) [inline tiflash-620-upgrade-guide.md:17]:
```suggestion
> - v6.2.0 新增了一项名为 [FastScan](/develop/dev-guide-use-fastscan.md) 的实验功能，该功能在 v7.0 GA。FastScan在牺牲强一致性保证的前提下可以大幅提升扫表性能。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: tiflash/use-fastscan.md [hunk 3]
```diff
@@ -1,14 +1,10 @@
 ---
-title: FastScan
+title: 使用 FastScan 模式
 summary: 介绍通过使用 FastScan 来加速 OLAP 场景的查询的方法。
 aliases: ['/zh/tidb/dev/sql-statement-set-tiflash-mode']
```
Comment (shichun-0415) [inline tiflash/use-fastscan.md:4]:
```suggestion
aliases: ['/zh/tidb/dev/sql-statement-set-tiflash-mode/','/zh/tidb/dev/dev-guide-use-fastscan/']
```

File: tiflash-620-upgrade-guide.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ summary: 了解升级 TiFlash 至 v6.2 时的注意事项。
 
 > **注意：**
 >
-> - v6.2.0 新增了一项名为 [FastScan](/develop/dev-guide-use-fastscan.md) 的实验功能。该实验功能在牺牲强一致性保证的前提下可以大幅提升扫表性能。相关实验功能的形态和使用方式有可能在后续版本中发生变化。
+> - v6.2.0 新增了一项名为 [FastScan](/tiflash/use-fastscan.md) 的实验功能，该功能在 v7.0 GA。FastScan在牺牲强一致性保证的前提下可以大幅提升扫表性能。
```
Comment (Oreoxmt) [inline tiflash-620-upgrade-guide.md:17]:
```suggestion
> - v6.2.0 新增了一项名为 [FastScan](/tiflash/use-fastscan.md) 的实验功能，该功能在 v7.0.0 GA。FastScan在牺牲强一致性保证的前提下可以大幅提升扫表性能。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by flowbehappy — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: TOC.md [hunk 5]
```diff
@@ -611,6 +610,7 @@
     - [使用 TiDB 读取 TiFlash](/tiflash/use-tidb-to-read-tiflash.md)
     - [使用 TiSpark 读取 TiFlash](/tiflash/use-tispark-to-read-tiflash.md)
     - [使用 MPP 模式](/tiflash/use-tiflash-mpp-mode.md)
+    - [使用 FastScan 模式](/tiflash/use-fastscan.md)
```
Comment (joey-yez) [inline TOC.md:613]:
```suggestion
    - [使用 FastScan 功能](/tiflash/use-fastscan.md)
```

File: tiflash/use-fastscan.md [hunk 6]
```diff
@@ -1,14 +1,10 @@
 ---
-title: FastScan
+title: 使用 FastScan 模式
```
Comment (joey-yez) [inline tiflash/use-fastscan.md:2]:
```suggestion
title: 使用 FastScan 功能
```

File: tiflash/use-fastscan.md [hunk 7]
```diff
@@ -1,14 +1,10 @@
...
-aliases: ['/zh/tidb/dev/sql-statement-set-tiflash-mode']
+aliases: ['/zh/tidb/dev/sql-statement-set-tiflash-mode/','/zh/tidb/dev/dev-guide-use-fastscan/']
 ---
 
-# FastScan
-
-> **警告：**
->
-> 该功能目前是实验性功能，其形式和使用方法可能会在未来版本中发生变化。
+# 使用 FastScan 模式
```
Comment (joey-yez) [inline tiflash/use-fastscan.md:7]:
```suggestion
# 使用 FastScan 功能
```

**Pull request merged**